auth/assoc-1.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: infoblox.com/v1
```

Comment:
can all of these PARGs be in a single file? or can they be named better? `assoc-1`,  `group-1` doesn't do me any good as a dumb lazy human when it comes to trying to figure out what it does.

Reply:
Maybe @johnbelamaric can weigh in here. I'm under the impression that each PARG should live in its own `.yaml` file.

Reply:
The are separate resources but they can be in the same file. And agree names should be meaningful.